zap/src/main/java/org/parosproxy/paros/security/SslCertificateServiceImpl.java

```diff
@@ -156,12 +158,13 @@ public KeyStore createCertForHost(CertData certData)
         namebld.addRDN(BCStyle.C, "xx");
         namebld.addRDN(BCStyle.EmailAddress, "owasp-zed-attack-proxy@lists.owasp.org");
 
+        long currentTime = System.currentTimeMillis();
         X509v3CertificateBuilder certGen =
                 new JcaX509v3CertificateBuilder(
                         new X509CertificateHolder(caCert.getEncoded()).getSubject(),
                         BigInteger.valueOf(serial.getAndIncrement()),
-                        new Date(System.currentTimeMillis() - Duration.ofDays(30).toMillis()),
-                        new Date(System.currentTimeMillis() + Duration.ofDays(1000).toMillis()),
+                        new Date(currentTime - Duration.ofDays(30).toMillis()),
+                        new Date(currentTime + Duration.ofDays(825 - 30).toMillis()),
```

Comment:
Worth extracting a constant (the JavaDoc could reference the decision also).

Reply:
Will do.